build.zig

```diff
@@ -31,7 +29,7 @@ pub fn build(b: *std.Build) void {
     // Creates a step for unit testing. This only builds the test executable
     // but does not run it.
     const main_tests = b.addTest(.{
-        .root_source_file = .{ .path = package_path },
+        .root_source_file = main_module.source_file,
```

Comment:
Ahhh, good to know this is how package exports should be done; still getting used to the new package manager! :)

Reply:
To be honest I don't know what „a proper way” is (there is no official guide unfortunately), I just noticed that they both duplicate same `LazyPath`